static/src/javascripts/bootstraps/enhanced/atoms.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import config from 'lib/config';
+import { services } from 'journalism/services';
+
+const bootstrapAtom = (atomMaker: AtomMaker, atomType: AtomType) => {
+    const atomBuilder = atomMaker[atomType].default(services);
+    [
+        ...document.querySelectorAll(`[data-atom-type='${atomType}']`),
+    ].forEach(atomDom => {
+        const atom = atomBuilder(atomDom).runTry();
+        if (typeof atom === 'string') {
+            // eslint-disable-next-line no-console
+            console.log(
+                `Failed to initialise atom [${atomType}/${atomDom.getAttribute(
+                    'data-atom-id'
+                ) || ''}]: ${atom}`
+            );
+        } else {
+            atom.start();
+        }
+    });
+};
+
+const initAtoms = () => {
+    if (config.get('page.atomTypes.guide')) {
```

Comment:
if `atomTypes` was an array, would we be able to drop all these and just import the modules for each type as we looped through?

Reply:
I don't know enough about the Webpack stuff, I agree there is opportunity for abstraction here.

Reply:
Dynamic imports are not supported by Webpack. Partially dynamic imports are supported, but Webpack will bundle all possible matches together. This would mean loading JS/CSS for atoms that may not be present on the page. 

Of course, this approach is verbose and possibly a premature optimisation. It would take a little experimentation / investigation